lib/features/modeling/Modeling.js

```diff
@@ -78,6 +78,8 @@ Modeling.prototype.getHandlers = function() {
     'connection.reconnectStart': ReconnectConnectionHandler,
```

Comment:
Please consider whether to remove these handlers. We do not use them anymore, right?